### PR TITLE
Various consistency changes

### PR DESCRIPTION
**Pull request by wezrule**
`migrate_working_path` can be removed as we only upgrade from v19 nodes now.
`nano::collect_container_info` should should east const ref for the `name` parameter
Moved `bootstrap_attempt_legacy` class to its own file, as `boostrap_attempt_lazy` has its own
Removed `~frontier_req_client ();` destructor as it's empty (and unnecessary)

### CODE REVIEW

**Review by argakiig — DISMISSED** [review]

File: nano/nano_node/entry.cpp [hunk 1]
```diff
@@ -65,6 +65,7 @@ class address_library_pair
 int main (int argc, char * const * argv)
 {
 	nano::set_umask ();
+	//
```
Comment (clemahieu) [inline nano/nano_node/entry.cpp]: Unused comment.

File: nano/node/lmdb/lmdb.cpp [hunk 2]
```diff
@@ -269,6 +269,7 @@ bool nano::mdb_store::do_upgrades (nano::write_transaction & transaction_a, bool
 		case 17:
 			upgrade_v17_to_v18 (transaction_a);
 			needs_vacuuming = true;
+			// Upgrades to v19 & v20 are both part of the v22 node release
```
Comment (clemahieu) [inline nano/node/lmdb/lmdb.cpp]: Should we start using [[fallthrough]]; ?

**Review by clemahieu — DISMISSED** [review]

File: nano/nano_node/entry.cpp (hunk 1 above)
Comment (wezrule) [reply nano/nano_node/entry.cpp]: Removed

File: nano/node/lmdb/lmdb.cpp (hunk 2 above)
Comment (wezrule) [reply nano/node/lmdb/lmdb.cpp]: Yeh we should, added

**Review by argakiig — APPROVED** [review]

**Pull request merged**